lib/Connection.js

```diff
@@ -799,11 +799,14 @@ Connection.prototype._fetch = function(which, uids, options) {
     fetching.push('X-GM-LABELS');
   }
   if (this.serverSupports('CONDSTORE') && !this._box.nomodseq)
-    fetching.push('MODSEQ');
-
-  fetching.push('UID');
-  fetching.push('FLAGS');
-  fetching.push('INTERNALDATE');
+    if(!options || options.modseq===undefined || options.modseq === true)
```

Comment:
This particular conditional is probably best folded into the one that checks for server support and such.

Additionally, each of these new conditionals (the one above and the others below) being added can be better simplified to something like:

```javascript
if (!options || options.modseq !== false)
```